pennylane_lightning/lightning_gpu/lightning_gpu.toml

```diff
@@ -58,7 +58,7 @@ MultiControlledX       = {}
 [operators.gates.matrix]
 
 BlockEncode            = {properties = [ "controllable" ]}
-GlobalPhase            = {properties = [ "controllable" ]}
+GlobalPhase            = {}
```

Comment:
We don't decompose to its matrix anymore. This can be removed.. 
```suggestion
```

Reply:
Nice catch